packages/core/contracts/financial-templates/optimistic-distributor/OptimisticDistributor.sol

```diff
@@ -67,9 +67,6 @@ contract OptimisticDistributor is Lockable, MultiCaller, Testable {
     uint256 public constant MINIMUM_LIVENESS = 10 minutes;
     uint256 public constant MAXIMUM_LIVENESS = 5200 weeks;
 
-    // Final fee can be synced and stored in the contract.
-    uint256 public finalFee;
```

Comment:
Agree, initially I did not fully realize OO methods return total bond amount that makes syncing final fee redundant.